### PR TITLE
Added XBee SX 868 Example Config

### PR DESCRIPTION
**Pull request by rodeo-sam**

### CODE REVIEW

File: common/source/docs/common-telemetry-xbee.rst [hunk 1]
```diff
@@ -45,71 +45,99 @@ restarted).
...
-|                   |RP=28                                       |RP=28                                        |
-+-------------------+--------------------------------------------+---------------------------------------------+
-| I/O Sampling      |IR=0,IC=0,V+=0                              |IR=0,IC=0,V+=0                               |
-+-------------------+--------------------------------------------+---------------------------------------------+
+application can be used to configure the XBee modules.
 
+Xbee ZB (S2C)
+=============
+
+Settings for a
```
Comment (Hwurzburg) [inline common/source/docs/common-telemetry-xbee.rst:53]: Settings for a what?

File: common/source/docs/common-telemetry-xbee.rst [hunk 2]
```diff
@@ -45,71 +45,99 @@ restarted).
...
++-------------------+-----------------------------------------------+-----------------------------------------------+
+| I/O Sampling      |IR=0, IC=0, V+=0                               |IR=0, IC=0, V+=0                               |
++-------------------+-----------------------------------------------+-----------------------------------------------+
+
+
+Example for XBee SX 868 (based on Firmware A00A).
+Everything is kept at default except the listed settings.
+Make sure to set the baud rate of the used telemetry port to 230400 and disable
+the rtscts setting as described above.
+
```
Comment (Hwurzburg) [inline common/source/docs/common-telemetry-xbee.rst:96]: what is this table for? how is it different than the one above? very confusing

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
commits should be squashed to one commit also

**Comment by rodeo-sam** [comment]
I will squash all commits upon merging.

**Comment by Hwurzburg** [comment]
thet need to be squashed to one commit before I will merge

**Comment by rodeo-sam** [comment]
see #6361 
this request can be closed then

**Pull request closed without merging**